cpu/avr/radio/rf230bb/halbb.c

```diff
@@ -667,7 +667,7 @@ ISR(TRX24_RX_START_vect)
 //	DEBUGFLOW('3');
 /* Save RSSI for this packet if not in extended mode, scaling to 1dB resolution */
 #if !RF230_CONF_AUTOACK
-    rf230_last_rssi = 3 * hal_subregister_read(SR_RSSI);
+    rf230_last_rssi = 90 - 3 * hal_subregister_read(SR_RSSI);
```

Comment:
instead of using "90" we could use a define #RSSI_BASE_VAL - like in the chip specification
but where to put it ? in rf230bb.h ? is it common for all rf230 ?